packages/legacy-scripting-runner/index.js

```diff
@@ -672,6 +672,7 @@ const legacyScriptingRunner = (Zap, zcli, input) => {
       request.headers = cleanHeaders(request.headers);
       request.allowGetBody = true;
       request.serializeValueForCurlies = serializeValueForCurlies;
+      request.skipThrowForStatus = true;
```

Comment:
This makes sure that even if a user deletes what https://github.com/zapier/zapier/pull/39898 adds to `afterResponse` for legacy (visual builder) apps, we are still in control of when we call `throwForStatus`.